chapters/type_table.txt

```diff
@@ -448,6 +448,22 @@
 |`int32_t[8]`
 |four dimensional 32 bit integer box (lower and upper bound vector)
 
+|`UINT64_BOX1`
+|`uint64_t[2]`
+|one dimensional 64 bit unsigned integer box (lower and upper bounds)
```

Comment:
how about clarifying the bounds here (as well or only here), i.e. `...box (inclusive lower and exclusive upper bounds)`?

Reply:
Then we should use a different type name, IMO: something like `UINT64_ELEMENT_RANGE`. We allow empty bounds (empty `FLOAT32_BOX3`, that is), so it's weird to me to state that the type disallows this in any context it possibly could be used in.